src/backends/kimchi/mod.rs

```diff
@@ -546,4 +551,159 @@ impl Backend for KimchiVesta {
 
         res
     }
+
+    fn constraint_neg(&mut self, var: &CellVar, span: Span) -> CellVar {
```

Comment:
remove `constraint` from the name (so just `neg`) as it wrongly imply that we will create a constraint here (same for others)